### PR TITLE
Fix the Dupe Perf Data Issue from the DaemonSet

### PR DESCRIPTION
**Pull request by r-dilip**

### CODE REVIEW

**Comment by r-dilip** [comment]
this change will leave a log line in the omsagent.log like:
**[warn]: no match for kubehealth.DaemonSet.Node** 
every so often (I observed it exponentially backing off, i.e every 1min, then every 2min, then every 4min, ....)

File: installer/conf/container-health.conf [hunk 1]
```diff
@@ -68,7 +68,7 @@
 </match>
 
 
-<match oms.api.KubeHealth.DaemonSet**>
+<match kubehealth.DaemonSet**>
```
Comment (vishiy) [inline installer/conf/container-health.conf]: when you go to IP discovery route (ignite changes i guess) change the below host in out_forward to IP as well)

File: installer/conf/kube.conf [hunk 2]
```diff
@@ -55,7 +55,7 @@
 #Kubernetes health
 <source>
   type kubehealth
- tag oms.api.KubeHealth.ReplicaSet
+ tag kubehealth.ReplicaSet
```
Comment (vishiy) [inline installer/conf/kube.conf]: kube.conf changes requires change in omsdocker/.yaml (for aks-engine) and the chart (for AKS). Please ensure you change both as part of roll out.

Comment (r-dilip) [reply installer/conf/kube.conf]: Yes. that is the plan

**Pull request merged**